### PR TITLE
Fix perftest history path

### PR DESCRIPTION
**Pull request by havogt**

### CODE REVIEW

**Comment by havogt** [comment]
launch jenkins

**Comment by havogt** [comment]
launch mpitests

**Comment by havogt** [comment]
launch perftests

File: jenkins/config.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+echo "Running on host $(hostname)"
```
Comment (fthaler) [inline jenkins/config.sh:1]: To me, the split between `config.sh` and `setup.sh` looks rather random to me. What’s the idea here?

**Review by fthaler — CHANGES_REQUESTED** [review]

File: jenkins/config.sh (hunk 1 above)
Comment (havogt) [reply jenkins/config.sh:1]: The main idea was to remove the duplicated explicit passes in plot_full_comparison.sh and setup.sh. But in the normal cases plot_full_comparison is probably called from other scripts that already include setup.sh. Should it also be callable standalone?

Comment (fthaler) [reply jenkins/config.sh:1]: Looks like plot_full_comparison is actually called on its own. But so we probably can just source the full setup.sh also in there? Should not hurt I guess and then all config stuff is really inside one file.

**Comment by havogt** [comment]
launch perftests

**Comment by havogt** [comment]
launch jenkins

**Comment by havogt** [comment]
launch perftests

**Comment by havogt** [comment]
launch jenkins

**Review by fthaler — APPROVED** [review]

**Pull request merged**